package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "buttercup-website",
-  "version": "4.2.0",
+  "version": "4.2.1",
   "description": "Buttercup's Website",
   "scripts": {
-    "build": "next build && next export",
+    "build": "next build",
```

Comment:
This is needed for release, please don't change it :)

Reply:
There was a warning stating that it was _now_ unneeded (may have been needed previously) when "output: export" was present in next.config.js

I can remove it, but is it really still needed?